tests/nsrt_learning/test_option_learning.py

```diff
@@ -445,3 +445,27 @@ def test_kinematic_action_conversion():
         loaded_converter = pkl.load(f)
     reduced_action_arr3 = loaded_converter.env_to_reduced(env_action_arr2)
     assert np.allclose(reduced_action_arr, reduced_action_arr3)
+    # Test that open and closed finger values are properly converted.
```

Comment:
Doesn't seem to be explicit test for `reduced_to_env` with potentially different gripper values. Would be good to have.